guides/common/modules/proc_optimizing-performance-by-removing-nics-from-database.adoc

```diff
@@ -0,0 +1,31 @@
+[id="optimizing-performance-by-removing-nics-from-database_{context}"]
+= Optimizing performance by removing NICs from database
+
+Filter and exclude the connections using the `Exclude pattern for facts stored in {Project}` and  `Ignore interfaces with matching identifier` option.
+By default, these options are set to most common hypervisors.
+If you name the virtual interfaces differently, you can update this filter to use it according to your requirements.
+
+.Procedure
+
+. On the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+. Select *Facts*.
+. To filter out all interfaces starting with specific names, for example, `blu`, add `blu*` to the `Ignore interfaces with matching identifier` option.
+. To prevent databases from storing facts related to interfaces starting with specific names, for example, `blu`, add `blu*` to the `Exclude pattern for facts stored in {Project}` option.
++
+By default, it contains the same list as the `Ignore interfaces with matching identifier` option.
+You can override it based on the requirements.
+This filters out facts completely without storing them.
+
+. To remove matching facts from the database (as added in *Administer* > *Settings* > *Facts* > the `Exclude pattern for facts stored in {Project}` option), enter the following command:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake facts:clean
+----
+
```

Comment:
```suggestion
```

Reply:
Removing the + sign breaks the numbering part of the procedure. Hence not considering this.